tests/unit/ListPeopleAndRoles.spec.ts

```diff
@@ -136,6 +140,116 @@ const peopleAndRoles = [
   }
 ]
 
+const gpPeopleAndRoles = [
+  {
+    officer: {
+      id: '0',
+      firstName: 'Romeo',
+      lastName: 'Whitehead',
+      middleName: 'D',
+      organizationName: '',
+      partyType: 'person',
+      email: 'completing-party@example.com'
```

Comment:
I see differences between this and Create UI.

I created a separate `describe` for FM due to slightly different data. See https://github.com/bcgov/business-create-ui/blob/299527f5e6b0f353f0b16f0891eb2ce52d547f52/tests/unit/ListPeopleAndRoles.spec.ts#L249.

Reply:
There is for sure. I created a peopleAndRoles for the separate GP test suite below this.
Will expand in following commits as well.